db/migrate/20171127154854_add_discretion_applied_to_application.rb

```diff
@@ -0,0 +1,9 @@
+class AddDiscretionAppliedToApplication < ActiveRecord::Migration
+  def up
+    add_column :details, :discretion_applied, :boolean
```

Comment:
Should this have default value? Maybe to avoid null values?

Reply:
@leoapost nil means that there was no discretion applied. true means it was granted and false it was denied. so you need to test 3 possibilities.

Reply:
oh, ok. Thanks for the clarification.